openslides_backend/models/motion_category.py

```diff
@@ -0,0 +1,32 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionCategory(Model):
+    """
+    Model for motion category.
+
+    Reverse fields:
+    - child_ids
+    - motion_ids
+    """
+
+    collection = Collection("motion_category")
+    verbose_name = "motion_category"
+
+    id = fields.IdField(description="The id of this motion category.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this motion_category.",
+        to=Collection("meeting"),
+        related_name="category_ids",
+    )
+    name = fields.RequiredCharField(description="The name of this motion category.")
+    prefix = fields.CharField(description="The prefix of this motion category.")
```

Comment:
Add `maxlength=32`

Reply:
Char Fields don't have a option maxlength.

Reply:
Ah, my bad, will come with #75. Not that important though, just leave it as is for now

Reply:
You can now use all constaints provided by jsonschema since the just get piped through in the `fields.py`. See https://json-schema.org/understanding-json-schema/reference/string.html#length for `maxLength`.